src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/PostTaskAssignByIdControllerTest.java

```diff
@@ -226,102 +228,6 @@ void should_return_500_with_application_problem_response_when_assign_call_fails(
             ));
     }
 
-    @ParameterizedTest
-    @CsvSource(value = {
-        "IA, Asylum",
-        "WA, WaCaseType"
-    })
-    void should_return_500_with_application_problem_response_when_assign_call_fails_due_to_missing_assignee_id(
```

Comment:
This is specifically testing 500 edge case when assignee id is not provided during Assign action. Is this removed for another reason?